spec/person_spec.rb

```diff
@@ -0,0 +1,26 @@
+require './person'
+require './rental'
+require './book'
+require 'rspec'
+
+RSpec.describe Person do
+  describe '#initialize' do
+    it 'should initialize a person with a name and age' do
+      person = Person.new('Rustamjon', 20)
+      expect(person.name).to eq('Rustamjon')
+      expect(person.age).to eq(20)
+    end
+    it 'should initialize a person with a name and age and parent permission' do
+      person = Person.new('Rustamjon', 20, { parent_permission: true })
+      expect(person.name).to eq('Rustamjon')
+      expect(person.age).to eq(20)
+      expect(person.parent_permission).to eq({ parent_permission: true })
+    end
+  end
+  describe '#rentals' do
+    it 'should return an empty array' do
+      person = Person.new('Rustamjon', 20)
+      expect(person.rentals).to eq([])
+    end
+  end
```

Comment:
- [ ] I would recommend adding more tests for `adding a new rental` by making instances of `book` and `person` and assigning it to rentals.

Reply:
Thanks for reminding this issue!